src/main/java/org/cubeville/cvstats/leaderboards/Leaderboard.java

```diff
@@ -250,6 +250,14 @@ private List<String> getLeaderboardLines() {
                 String value = leaderboardResults.getString("value");
                 if (this.key.equals("player")) {
                     key = Bukkit.getOfflinePlayer(UUID.fromString(key)).getName();
+                } else if (this.key.equals("players")) {
+                    List<String> playerNames = new ArrayList<>();
+                    for (String player : key.split(",")) {
+                        playerNames.add(Bukkit.getOfflinePlayer(UUID.fromString(player)).getName());
+                    }
+                    StringJoiner joiner = new StringJoiner(", ");
+                    playerNames.forEach(joiner::add);
+                    key = joiner.toString();
```

Comment:
Could you add a sorting mechanism so that the players are ordered in the same way every time? (Maybe by sorting their UUIDs?) Otherwise, I anticipate a situation where we have a bunch of duplicate inputs for a set of players otherwise, and it'd be nice to get data based on a certain group.

Reply:
> Could you add a sorting mechanism so that the players are ordered in the same way every time? (Maybe by sorting their UUIDs?) Otherwise, I anticipate a situation where we have a bunch of duplicate inputs for a set of players otherwise, and it'd be nice to get data based on a certain group.

I was doing that on the plugin-side implementing into it, but I suppose it would make sense to do it on this side either instead or in addition to the plugin implementing into it.

I hadn't considered that player appears in other places, my bad. I don't know why I didn't think to check. I'll work on this.

Reply:
No problem! That's what PRs are for ^^